platformio.ini

```diff
@@ -19,4 +19,5 @@ lib_deps =
 	feilipu/FreeRTOS@^11.0.1-5
 	mprograms/SimpleRotary@^1.1.3
 	madleech/Button@^1.0.0
-	Wire
\ No newline at end of file
+	Wire
+	https://github.com/mpaland/printf.git#v4.0.0
```

Comment:
Examples won't compile unless you add this dependency in the `compile-arduino.yml` workflow